lib/galaxy/config/sample/job_metrics_conf.xml.sample

```diff
@@ -47,6 +47,10 @@
   <!-- <cgroup verbose="true" /> -->
```

Comment:
A [few lines above this one](https://github.com/galaxyproject/galaxy/pull/17169/files#diff-52015c067f57c59cca7752f8e10fdd50ff69df89cf3b7d61ca4179487cc495faR43) (I cannot comment there), the path "lib/galaxy/jobs/metrics/instrumenters/cgroup.py" is wrong, it should be "lib/galaxy/job_metrics/instrumenters/cgroup.py".